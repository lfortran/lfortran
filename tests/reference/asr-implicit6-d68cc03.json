{
    "basename": "asr-implicit6-d68cc03",
    "cmd": "lfortran --indent --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/implicit6.f90",
    "infile_hash": "ec52eac2f61928998941f6ef5395e23263fedec17cbec758e10e24ef",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-implicit6-d68cc03.stdout",
    "stdout_hash": "5198f9e044e7324404b6edbae665dd54af890f6d3c316f0735619429",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}