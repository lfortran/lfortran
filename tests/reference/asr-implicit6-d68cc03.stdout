(TranslationUnit
    (SymbolTable
        1
        {
            ch12n:
                (Function
                    (SymbolTable
                        2
                        {
                            ch12n:
                                (Variable
                                    2
                                    ch12n
                                    []
                                    ReturnVar
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            ci:
                                (Variable
                                    2
                                    ci
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            n:
                                (Variable
                                    2
                                    n
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            z:
                                (Variable
                                    2
                                    z
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    ch12n
                    (FunctionType
                        [(Integer 4 [])
                        (Real 4 [])]
                        (Integer 4 [])
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    []
                    [(Var 2 n)
                    (Var 2 z)]
                    [(=
                        (Var 2 ci)
                        (Cast
                            (ComplexConstructor
                                (RealConstant
                                    0.000000
                                    (Real 8 [])
                                )
                                (RealConstant
                                    1.000000
                                    (Real 8 [])
                                )
                                (Complex 8 [])
                                (ComplexConstant
                                    0.000000
                                    1.000000
                                    (Complex 8 [])
                                )
                            )
                            ComplexToReal
                            (Real 4 [])
                            (RealConstant
                                0.000000
                                (Real 4 [])
                            )
                        )
                        ()
                    )]
                    (Var 2 ch12n)
                    Public
                    .false.
                    .false.
                )
        })
    []
)
